### PR TITLE
Unlimited room capacity

### PR DESCRIPTION
**Pull request by Lorax66**
Fix issue #5

### CODE REVIEW

**Review by felixblanke — COMMENTED** [review]
Changes LGTM. But wouldn't it be more efficient to simply avoid checking the room constraints on rooms of infinite size?

**Comment by felixblanke** [comment]
We might also want to add a remark in the wiki how unlimited capacities are handled in the end.

**Comment by Lorax66** [comment]
Good point, we can add this once #8 is merged. Adding this now would lead to conflicts.

**Comment by Lorax66** [comment]
Done, we now only add a constraint for the room size if the room would not fit the whole AK.

File: src/akplan/solve.py [hunk 1]
```diff
@@ -51,6 +51,30 @@ def process_pref_score(preference_score: int, required: bool, mu: float) -> floa
...
+
+    Raises:
+        ValueError: if 'room_capacity' < -1
+    """
+    if room_capacity == -1:
+        return num_participants
+    if room_capacity >= num_participants:
+        return num_participants
+    if room_capacity < 0:
+        raise ValueError(room_capacity)
```
Comment (felixblanke) [inline src/akplan/solve.py:74]:
```suggestion
        raise ValueError(
            f"Invalid room capacity {room_capacity}. "
            "Room capacity must be non-negative or -1."
        )
```

**Review by felixblanke — APPROVED** [review]
LGTM

**Pull request merged**